src/utils/git.js

```diff
@@ -182,7 +182,7 @@ const getRefs = async () => {
     'for-each-ref',
     `--sort=${conf.get('sort')}`,
     '--format=%(refname)',
-    '--count=500',
+    `--count=${conf.get('refCount')}`,
```

Comment:
To build on the comment about not changing the config object. When a user passes the option to override the count, i think we can use that over the options only when present, otherwise we can use the default options.

Reply:
modified accordingly.